test/lib/usd/translators/testUsdExportSkeleton.py

```diff
@@ -205,6 +207,49 @@ def testSkelWithoutBindPose(self):
             animSource.GetTranslationsAttr().Get(5.0),
             Vt.Vec3fArray([Gf.Vec3f(5.0, 5.0, 0.0)]))
 
+    def testSkelRestXformsWithNoDagPose(self):
+        """
+        Tests export of of rest xforms when there is no dagPose node at all.
```

Comment:
typo: `of of`

Reply:
Fixed in https://github.com/Autodesk/maya-usd/pull/1130/commits/afaea5505a8aadef0d02bff165e72cc36e5c8a36